keyboards/treasure/type30/hotswap/config.h

```diff
@@ -0,0 +1,56 @@
+// TREASURE TYPE 30
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+/* disable action features */
+
+#define NO_ACTION_TAPPING
+#define NO_ACTION_ONESHOT
+
+#define RGB_MATRIX_LED_COUNT 36
+#define DRIVER_LED_TOTAL 36
```

Comment:
These shouldn't be needed, as the json file will generate it based on the number of LEDs there.
```suggestion
```